log4j-ecs-layout/README.md

```diff
@@ -5,12 +5,30 @@
 Latest version: [![Maven Central](https://img.shields.io/maven-central/v/co.elastic.logging/log4j-ecs-layout.svg)](https://search.maven.org/search?q=g:co.elastic.logging%20AND%20a:log4j-ecs-layout)
 
 Add a dependency to your application
+
 ```xml
 <dependency>
     <groupId>co.elastic.logging</groupId>
     <artifactId>log4j-ecs-layout</artifactId>
     <version>${ecs-logging-java.version}</version>
 </dependency>
+
+<dependency>
+    <groupId>co.elastic.logging</groupId>
+    <artifactId>ecs-logging-core</artifactId>
+    <version>${ecs-logging-java.version}</version>
+ </dependency>
```

Comment:
Adding this dependency explicitly is not required as `log4j-ecs-layout` has a transitive dependency on it
```suggestion
```